pinot-tools/src/test/resources/memory_estimation/schema.json

```diff
@@ -0,0 +1,36 @@
+{
+  "schemaName": "testTable",
+  "dimensionFieldSpecs": [
+    {
```

Comment:
Let's make this as complete as possible. 

All fixed width - both single and multi-value
All variable width (string and bytes) - both single and multi value

Reply:
Also, if you think about this from integration perspective, RecommenderDriver doesn't take schema. It takes the schema annotated with characteristics.  (See the files RecommenderDriver, InputManager and SchemaWithMetadata). So, something to think about for integration. If now you are providing MemoryEstimator with schema separately then I am not sure if you are going to change it back to the annotated schema in the follow-up PR when you will integrate with RecommenderDriver.

Reply:
Okay, I gave up 😃  (To be fair, I wanted to do that in the next PR which is the integration part, but now it has it!)
Refactored the code to use `SchemaWithMetadata`. Now only one file is provided and that one represents both schema and data characteristics.